### PR TITLE
fix graph missing stop gradient layers

### PR DESCRIPTION
**Pull request by wzhongyuan**
## What changes were proposed in this pull request?
Graph missing stop gradient layers after deserialization

## How was this patch tested?

unit test

### CODE REVIEW

**Comment by wzhongyuan** [comment]
@yiheng  please help review, thanks

**Comment by wzhongyuan** [comment]
PR validation passed https://github.com/intel-analytics/spark-dl/pull/2246

File: spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/Graph.scala [hunk 1]
```diff
@@ -599,13 +601,25 @@ trait GraphSerializable extends ContainerSerializable {
     (implicit ev: TensorNumeric[T]): AbstractModule[Activity, Activity, T] = {
     val (module, inputs, outputs, generateBackwardValue, sharedVariables) =
       prepareLoadModule(context)
-    if (generateBackwardValue != null) {
+    val attributes = module.getAttrMap
```
Comment (yiheng) [inline spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/Graph.scala]: looks like this line is useless

**Comment by yiheng** [comment]
others look good to me

File: spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/Graph.scala (hunk 1 above)
Comment (wzhongyuan) [reply spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/Graph.scala]: it's used in line 615 below, so shall I merge it ?

**Comment by wzhongyuan** [comment]
Closing it now as PR passed

**Pull request merged**